scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -26,8 +26,8 @@ EOF
 # the amzn2-nvidia repo is temporary and only used for installing the system-release-nvidia package
 sudo mv $tmpfile /etc/yum.repos.d/amzn2-nvidia-tmp.repo
 
-# only install open driver for post-kepler gpus
-if [[ $AMI_TYPE != "al2keplergpu" ]]; then
+# only install open driver for post-kepler gpus, exclude airgapped regions
+if [[ $AMI_TYPE != "al2keplergpu" && -z ${AIR_GAPPED} ]]; then
```

Comment:
nit: any reason we can't do `if [ -n "$AIR_GAPPED" ]; then...` like we do on L11? Or we could check if `GPG_CHECK = 0` as well.